templates/member/settings/github.html

```diff
@@ -47,7 +47,8 @@
 
     <p>
         {% trans "Remplissez le formulaire suivant pour enregistrer votre token GitHub afin de pouvoir créer des tickets pour les sujets. Vous pouvez créer un token sur" %}
-        <a href="https://github.com/settings/tokens">{% trans "cette page" %}</a>.
+        <a href="https://github.com/settings/tokens">{% trans "cette page" %}</a>. Le token doit avoir au moins le scope
+        <kbd></kb><samp>public_repo</samp></kbd>.
```

Comment:
```suggestion
        <code>public_repo</code>.
```

Sans compter que `kbd` est fermé deux fois, la sémantique des balises n'est pas bonne.
- `kbd` représente une saisie clavier lettre par lettre, genre `<kbd>Ctrl</kbd> + <kbd>Maj</kbd> + <kbd>R</kbd>`. Ce n'est pas le cas ici.
- `samp` _pourrait_ passer, ça représente la sortie d'un programme. C'est pas vraiment ça non plus mais ça peut se tenir.

Dans ce cas j'utiliserais plutôt `code`, qui est le plus proche sémantiquement parlant.

Reply:
Je suis ça en fait : https://developer.mozilla.org/fr/docs/Web/HTML/Element/kbd#representer_les_options_de_saisies_a_l_ecran (aux erreurs de fermeture de balise près)

Reply:
Bon, du coup on fait quoi au final ? :D

Reply:
Pour moi, ce que j'ai fait est correct car recommandé par la doc de Mozilla. Mais c'est mon code, alors je ne me permets pas de m'autovalider. ^^